### DISCUSSION TITLE
How to upgrade ABP 5.3 to ABP 8.1?

### DISCUSSION THREAD

**Discussion by vishnukumarspeehive** [abpframework/abp#19560] [Q&A]
I am currently working with an ABP Framework project template version 5.3, utilizing .NET 6. Now, I aim to upgrade it to the latest ABP Framework template, version 8.1, while also migrating to .NET 8. My project follows a Layered architecture.

Additionally, my frontend is developed using Angular, with a version of "~13.1.1".

I would appreciate guidance on the steps required for this update process. Specifically, I am unsure whether there exists a direct update option to version 8.1 or if I should follow a sequential update path, such as moving from 5 to 6, then 6 to 7, and finally reaching 7.8 before updating to 8.1.

Could someone advise on the best approach for this scenario?

**Comment by realLiangshiwei** [comment]
See https://docs.abp.io/en/abp/latest/Migration-Guides/Index
